### DISCUSSION TITLE
文本切分过短

### DISCUSSION THREAD

**Discussion by BUGLI27** [RapidAI/RapidRAG#21] [Q&A] [answered]
sentence_size为200，发现检索返回的结果长度都很短，ChineseTextSplitter切分方式是否需要进一步优化？
```yaml
# text splitter
SENTENCE_SIZE: 200
```
top15检索结果如下
<img width="702" alt="image" src="https://github.com/RapidAI/Knowledge-QA-LLM/assets/87059873/937e9be7-3757-41ff-a2d7-8dcafc431e55">

**Comment by SWHL** [comment, marked as answer]
`SENTENCE_SIZE `参数是当当前语句长度大于200时，才会触发文本切分。
从你给的结果来看，文档正常读取后长度就是这么短。

**Reply by BUGLI27** [reply]
明白了，谢谢！还以为是分成每段长度在200左右的句子。
